### PR TITLE
Implemented an upload script

### PR DESCRIPTION
**Pull request by mehrdadhalali**
The code can do with some refactoring, but it works.

### CODE REVIEW

File: load_to_rds/upload_to_db.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+    upload_all_listings_to_database(gog_data, maps, conn)
+    upload_all_listings_to_database(steam_data, maps, conn)
+
+    conn.close()
+
+
+if __name__ == "__main__":
+
+    main()
```
Comment (mehrdadhalali) [inline load_to_rds/upload_to_db.py:152]: Yes, I know this is a cardinal sin, I will change this.

File: load_to_rds/README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+Ensure that you have the following:
+- **Docker** installed for containerisation.
+- **AWS CLI** configured to interact with AWS services (ECR, Lambda, RDS, etc.)
+- **Python** installed on your local machine. 
+---
+
+## 📂 Setup
+
+1. Create a `.env` file with the following content:
```
Comment (Leon-JS02) [inline load_to_rds/README.md:17]: Very informative readme

File: schema/schema.sql [hunk 3]
```diff
@@ -58,10 +58,9 @@ FOREIGN KEY (os_id) REFERENCES operating_system(os_id)
 
 CREATE TABLE game_listing (
     listing_id INT GENERATED ALWAYS AS IDENTITY,
-    game_id INT NOT NULL UNIQUE ,
-    platform_id SMALLINT NOT NULL UNIQUE,
```
Comment (Leon-JS02) [inline schema/schema.sql:62]: Good spot!

**Review by Leon-JS02 — APPROVED** [review]
This is solid code. With a few refactors it'll be great.

File: load_to_rds/clear_database.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME -f delete.sql
```
Comment (paulauus) [inline load_to_rds/clear_database.sh:3]: So this is what has been messing with my data :p

**Review by paulauus — APPROVED** [review]

**Pull request merged**